packages/backend/server/src/__tests__/copilot.e2e.ts

```diff
@@ -747,16 +753,36 @@ test('should be able to manage context', async t => {
     );
   }
 
+  const fs = await import('node:fs');
+  const buffer = fs.readFileSync(
+    new URL('../../../../common/native/fixtures/sample.pdf', import.meta.url)
```

Comment:
```ts
import { Package }  from @affine-tools/utils/workspace

new Package('@affine/native').join('fixtures/sample.pdf')
```

Reply:
not avaliable for pure rust project

Reply:
common/native not have project.json

Reply:
then use `ProjectRoot` from '@affine-tools/utils/path'?

Reply:
lgtm